frontend/src/core/utils/fluent/getReconstructedMessage.js

```diff
@@ -31,7 +31,7 @@ export default function getReconstructedMessage(original: string, translation: s
             translation.split('\n').forEach(t => content += `\n        ${t}`);
         }
         else {
-            content = `${key} =\n    .${attribute} = ${translation}`;
+            content = `${key} = ${translation}\n    .${attribute} =`;
```

Comment:
That change is not correct. It creates an invalid Fluent message. There is no bug here, so you definitely shouldn't make any change of that sort.